content/2.developers/3.guides/5.fee-grant/3.utilize-grant.md

```diff
@@ -0,0 +1,283 @@
+---
+
+objectID: developers|guides|fee-grant|utilize-grant
+title: Utilizing grant allowances
+description: Instructions on how to utilize allowances assigned from the fee grant module
+parentSection: Developers
+parentSectionPath: /developers
+---
+
+#  Using grant allowances to execute transactions
+
+As one of the pimary use case of the Fee Grant module is to help with onboarding new users, this section of the guide will show how to allow users to execute transactions using the allowance granted to them.
+
+## Query a grant allowance
+
+Now that an allowance has been assigned to a account address, you should be able to verify the details of that grant. The returned information will include the granter's address, the allowance amount, and any other conditions set by the granter, such as the expiration date of the allowance. 
+
+The following command using archwayd will give the details of a grant allowance:
+
+```console
+archwayd query feegrant grant "${granter_address}" "${grantee_address}" --chain-id constantine-3 --node https://rpc.constantine.archway.tech:443
+```
+
+**Results**:
+
+```
+allowance:
+  '@type': /cosmos.feegrant.v1beta1.BasicAllowance
+  expiration: null
+  spend_limit:
+  - amount: "4806067100000000000"
+    denom: aconst
+grantee: archway1vve3d5u89h966t8kcvemwu88rnzzgcqxvkuj4s
+granter: archway1u0p8vfpf5gjwje84073x0d6w40vm2s25603359
+```
+
+This can also be achieved via the following API endpoint where **api_endpoint** is an API endpoint connected to one of the Archway networks (Testnet or Mainnet). **granter** is the address that granted the allowance and **grantee** is the address that received the allowance:
+
+```
+{api_endpoint}/cosmos/feegrant/v1beta1/allowance/{granter}/{grantee}
+```
+
+**Results**:
+
+```
+{
+  "allowance": {
+    "granter": "archway1u0p8vfpf5gjwje84073x0d6w40vm2s25603359",
+    "grantee": "archway1vve3d5u89h966t8kcvemwu88rnzzgcqxvkuj4s",
+    "allowance": {
+      "@type": "/cosmos.feegrant.v1beta1.BasicAllowance",
+      "spend_limit": [
+        {
+          "denom": "aconst",
+          "amount": "4806067100000000000"
+        }
+      ],
+      "expiration": null
+    }
+  }
+}
+```
+
+## Creating a Transaction
+
+Just like any other transaction on **Archway**, a transaction that will use a granted allowance for fees must include certain information, such as the type of transaction (i.e., the message). Instead of the normal fee object, the transaction should include the granter's account which will pay the transaction fee. <!--So instead of setting the **fee** value to **auto** we use the **calculateFee** method to dynamically generate the fee but also pluggin in the **feeGranterAddress** which is the address of the **granter** that will cover the fees.--> This is accomplished by doing the following:
```

Comment:
```suggestion
Just like any other transaction on **Archway**, a transaction that will use a granted allowance for fees must include certain information, such as the type of transaction (i.e., the message). Instead of the normal fee object, the transaction should include the granter's account which will pay the transaction fee. <!--So instead of setting the **fee** value to **auto** we use the **calculateFee** method to dynamically generate the fee, it should be added the **feeGranterAddress** which is the address of the **granter** that will cover the fees.--> This is accomplished by doing the following:
```